### PR TITLE
HW1

### PR DESCRIPTION
**Pull request by Sigul4**

### CODE REVIEW

File: students/Sigul4/hw1/HW1_Tsigulov.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        <div class="container">
+            <div class="slider-track"></div>
+            <input type="range" min="0" max="100" value="30" id="slider-1" oninput="slideOne()">
+            <input type="range" min="0" max="100" value="70" id="slider-2" oninput="slideTwo()">
+        </div>
+    </div>
+    
+     <!--Script-->
+     <script >
+        window.onload = function(){
```
Comment (Temu4) [inline students/Sigul4/hw1/HW1_Tsigulov.html:32]: Better practice is to run initialization scripts at the bottom of the file

File: students/Sigul4/hw1/HW1_Tsigulov.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            slideOne();
+            slideTwo();
+        }
+
+        let sliderOne = document.getElementById("slider-1");
```
Comment (Temu4) [inline students/Sigul4/hw1/HW1_Tsigulov.html:37]: When you need to get a DOM element use `const` instead `let`

File: students/Sigul4/hw1/HW1_Tsigulov.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            fillColor();
+        }
+        function fillColor(){
+            percent1 = (sliderOne.value / sliderMaxValue) * 100;
+            percent2 = (sliderTwo.value / sliderMaxValue) * 100;
+            sliderTrack.style.background = `linear-gradient(to right, #dadae5 ${percent1}% , #3264fe ${percent1}% , #3264fe ${percent2}%, #dadae5 ${percent2}%)`;
+        }
+    </script>
+</body>
+</html>
```
Comment (Temu4) [inline students/Sigul4/hw1/HW1_Tsigulov.html:66]: Please add a new line at the bottom of your files.
Why? [link](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)
How to? [link](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files)

File: students/Sigul4/hw1/style.css [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    height: 0;
+    width: 0;
+    border-top: 15px solid #3264fe;
+    border-left: 15px solid transparent;
+    border-right: 15px solid transparent;
+    margin: auto;
+    bottom: -14px;
+    left: 0;
+    right: 0;
+}
```
Comment (Temu4) [inline students/Sigul4/hw1/style.css:119]: Please add a new line at the bottom of your files.
Why? [link](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)
How to? [link](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files)

**Review by Temu4 — CHANGES_REQUESTED** [review]
Hi! Interesting interaction)

But there are still few issues.

And please remove an empty file `students/Sigul4/hw1/HW1_Tsigulov.js`

File: students/Sigul4/hw1/HW1_Tsigulov.html (hunk 2 above)
Comment (Sigul4) [reply students/Sigul4/hw1/HW1_Tsigulov.html]: Changes let to const

File: students/Sigul4/hw1/HW1_Tsigulov.html (hunk 3 above)
Comment (Sigul4) [reply students/Sigul4/hw1/HW1_Tsigulov.html]: New line added

File: students/Sigul4/hw1/style.css (hunk 4 above)
Comment (Sigul4) [reply students/Sigul4/hw1/style.css]: New line added

**Comment by Temu4** [comment]
@Sigul4 could you please be more precisely and call folder with correct name: `hw1`->`hw-1`

**Pull request closed without merging**